Description: Fix wrong entries in xref table.
 This fixes a off-by-one error in the included itext-paulo library which makes
 at least the burst command to produce non-standard-conform PDF documents
 whose free entries in the xref table are wrongly marked as used.
Author: Johann Felix Soden <johfel@debian.org>
Bug-Debian: http://bugs.debian.org/685983


--- a/java/com/lowagie/text/pdf/PdfWriter.java
+++ b/java/com/lowagie/text/pdf/PdfWriter.java
@@ -309,7 +309,7 @@
         
         int getIndirectReferenceNumber() {
             int n = refnum++;
-            xrefs.add(new PdfCrossReference(n, 0, 65536));
+            xrefs.add(new PdfCrossReference(n, 0, 65535));
             return n;
         }
         
